src/Ocelot/LoadBalancer/LoadBalancers/ServiceIsInvalidError.cs

```diff
@@ -0,0 +1,12 @@
+using Ocelot.Errors;
+
+namespace Ocelot.LoadBalancer.LoadBalancers
+{
+    public class ServiceIsInvalidError : Error

```

Comment:
Yes, I have seen that class and where it is used. It is used for errors when instead of a list of services, null is received (Even the name of the class is used in the plural form).
In this case, there will be a list, but one of the elements will be null, and the other will be valid.

But if you still think it's better to use it, then okay, I'll delete it.

Reply:
The error message can be interpreted as "Some services are null." 😊

Reply:
Deleted custom error and now using ServicesAreNullError

Reply:
Fixed in https://github.com/ThreeMammals/Ocelot/pull/2111/commits/bbed7d1aa195f425939cc5f3a69e245f6bdd565e ✔️